types/ua-parser-js/ua-parser-js-tests.ts

```diff
@@ -45,9 +50,21 @@ result.cpu.architecture; // $ExpectType string | undefined
 parser.getCPU().architecture; // $ExpectType string | undefined
 
 // Extensions
-const uaString = 'ownbrowser/1.3';
+const uaString = "ownbrowser/1.3";
 const ownBrowser = [[/(ownbrowser)\/([\w\.]+)/i], [UAParser.BROWSER.NAME, UAParser.BROWSER.VERSION]];
-parser = new UAParser(uaString, { browser: ownBrowser }); // $ExpectType UAParser
+new UAParser({ browser: ownBrowser }); // $ExpectType UAParserInstance
```

Comment:
@BendingBender Thank you for working on this!

I’m not sure if I’m reading this correctly, but wouldn’t this break backward compatibility with existing code like `const parser: UAParser = new UAParser()`?
Could for example `declare type UAParser = UAParserInstance;` coexist with `declare const UAParser` in .d.ts?

Reply:
Yeah, you're right, fixed this.